### PR TITLE
Removes coverage report from every test run

### PR DESCRIPTION
**Pull request by benjclark**
The coverage report always slows down tests and imo there's no need to run it every time, it would be better if we run it ad hoc?

### CODE REVIEW

**Comment by jpw** [comment]
As for the first thing, have you run the bootstrap command? https://github.com/springernature/frontend-toolkits#npm-scripts

(I hit this recently in another repo, perhaps it is time to rethink this approach, but not sure what we can do...?)

File: context/brand-context/package.json [hunk 1]
```diff
@@ -4,5 +4,8 @@
   "license": "MIT",
   "description": "Bootstrapping for all components and products",
   "keywords": [],
-  "author": "Springer Nature"
+  "author": "Springer Nature",
+  "devDependencies": {
```
Comment (tulipdexter) [inline context/brand-context/package.json:8]: I'm a bit confused, what is this for?

Comment (benjclark) [reply context/brand-context/package.json:8]: The Jest tests fail if the @springernature imports in the tests and actual modules are not installed in the package

**Comment by benjclark** [comment]
@jpw  Ah yes, thank you. I hadn't spotted that before. That's good to know. Thanks!

**Pull request closed without merging**